dbfit-java/core/src/main/java/dbfit/util/FitNesseTestHost.java

```diff
@@ -9,6 +10,7 @@
 
     static {
         TypeAdapter.registerParseDelegate(BigDecimal.class, BigDecimalParseDelegate.class);
+        TypeAdapter.registerParseDelegate(BigInteger.class, BigIntegerParseDelegate.class);
```

Comment:
In [`MySQL jdbc driver docs`](http://dev.mysql.com/doc/connector-j/en/connector-j-reference-type-conversions.html) it's mentioned that `BIGINT UNSIGNED` is mapped to `java.math.BitInteger`.